systers_portal/static/css/style.css

```diff
@@ -261,6 +265,13 @@ table.decoration-none  tr a {
   padding-left: 15px;
 }
 
+/* Meetup Location Members user-cell space
+-------------------------------------------------- */
+.user-cell-wh-100 {
+  width: 100px;
+  height: 100px;
```

Comment:
You are setting the height of 100px to a div that contains an image of height 100px + the username.
This is what happens
![image](https://cloud.githubusercontent.com/assets/2039122/9027889/a88f349c-395e-11e5-8cbc-b4c59ca44871.png)